pkg/controller-gen/args/args.go

```diff
@@ -12,6 +12,8 @@ type CustomArgs struct {
 	TypesByGroup  map[schema.GroupVersion][]*types.Name
 	Options       Options
 	OutputBase    string
+	// TODO: Rename?
+	Boilerplate []byte
```

Comment:
I'm gonna try to remove this to make it as it was before if possible.

Reply:
As discussed previously, I ended up renaming this to `BoilerplateContent` instead and added a comment.

Ideally, `CustomArgs` would be private and wouldn't embed `Options`, but that's not the case yet :shrug: